ocs_ci/ocs/ripsaw.py

```diff
@@ -88,6 +88,26 @@ def apply_crd(self, crd):
         run(f'oc apply -f {crd}', shell=True, check=True, cwd=self.dir)
         run(f'oc apply -f {self.operator}', shell=True, check=True, cwd=self.dir)
 
+    def get_uuid(self, benchmark):
+        """
+        Getting the UUID of the test.
+           when ripsaw used for running a benchmark tests, each run get its own
+           UUID, so the results in the elastic-search server can be sorted.
+
+        Args:
+            benchmark (str): the name of the main pod in the test
+
+        Return:
+            str: the UUID of the test
+        """
+        output = self.pod_obj.exec_oc_cmd(f'exec {benchmark} -- env')
+        uuid = ''
+        for line in output.split():
+            if 'uuid=' in line:
+                uuid = line.split('=')[1]
+        log.info(f'The UUID of the test is : {uuid}')
+        return uuid
```

Comment:
Should these lines be within the `if` statement or do we really want to keep iterating over the output after we have found the `uuid`? If these lines move into the `if` statement we can also remove `uuid = ''` from line 104

Reply:
I used this approach to make sure that the uuid is defined by the ripsaw benchmark operator.
If in future benchmarks run by the ripsaw will have a bug and this variable will not define, this test will not failed.